packages/react-components/react-divider/MIGRATION.md

```diff
@@ -1,60 +1,36 @@
 # Divider Migration
 
-## STATUS: WIP 🚧
-
-This Migration guide is a work in progress and is not yet ready for use.
-
 ## Migration from v8
 
-The existing `Separator` control supports the `content` property as well as children nodes. The `content` property has been removed and now only supports children.
+The existing v8's `Separator` control supports a very similar set of props to the one being proposed for v9's `Divider` with a few differences that are outlined below:
```

Comment:
```suggestion
The existing v8 `Separator` control supports a very similar set of props to the one being proposed for v9's `Divider` with a few differences that are outlined below:
```

Reply:
Or, since we're editing this sentence, it's probably also better to reword it to remove implications that the v9 Divider doesn't exist yet:
```suggestion
The v8 `Separator` control supports a very similar set of props to the v9 `Divider` with a few differences that are outlined below:
```